.babelrc

```diff
@@ -2,7 +2,13 @@
   "plugins": [
     "transform-decorators-legacy",
     "lodash",
-    "transform-runtime"
+    "transform-runtime",
+    [
```

Comment:
from my limited experience, we don't actually want the react-intl config in .babelrc. instead, we want a separate babel compile whose only purpose is to extract the language files. That said, my experience is roughly 60 minutes.